files/sharepointplus/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "sharepointplus",
+  "repo": "aymkdn/sharepointplus",
+  "files": {
+    "basePath": "jsdelivr/"
```

Comment:
Since we already host the unminified file here http://www.jsdelivr.com/projects/sharepointplus we need to add it to updates as well. Unless you release a major version, then we dont worry about breaking the file structure